app/javascript/lib/metrics-api.js

```diff
@@ -392,13 +392,21 @@ function getTabProperty() {
  * @param {Object} props
  */
 export function log(name, props = {}) {
+  let isDEEnabled // track DE globally
```

Comment:
Maybe it's overkill but maybe here too "track Differential Expression (DE) globally)